delta/utils/solver/utils/solver_utils.py

```diff
@@ -79,6 +83,66 @@ def save_infer_res(config, logits, preds):
       in_f.write(" ".join(["{:.3f}".format(num) for num in logit]) +
                  "\t{}\n".format(pred))
 
+def get_model_file(dir_name, file_name_pattern, mode, model_load_type, specified_model_file_name):
+  """Return model file according the specified model_load_type"""
+  if model_load_type is None:
+    logging.info("The values of model_load_type is not specified.")
+    model_load_type = "latest" if mode == utils.TRAIN else "best"
+    logging.info(
+        "For the {} command, model_load_type:{} is adopted.".format(mode, model_load_type))
+
+  #model_load_type can not be 'scratch' when performing EVAL or INFER command
+  if model_load_type == 'scratch' and mode != utils.TRAIN:
+    model_load_type = "best"
+    logging.info(
+        "The model_load_type cannot be scratch when performing {} command, and is changed to {}"
+        .format(mode, model_load_type))
+ 
+  assert model_load_type in ["best", "latest", "scratch", "specific"]
+
+  #get the path of model file according the specificed model_load_type
+  model_file_name = None
+  if model_load_type == "latest":
+    model_file_name = get_most_recently_modified_file_matching_pattern(dir_name, file_name_pattern)
+  elif model_load_type == "best":
+    model_file_name = Path(dir_name).joinpath('best_model.ckpt')
+  elif model_load_type == "specific":
+    if specified_model_file_name is None:
```

Comment:
using `latest` when `specified_model_file_name` is None, and logging the changes.